src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -95,6 +95,36 @@ private static IEnumerable<object[]> SerializableObjects()
             yield return new object[] { " < ", new string[] { "AAEAAAD/////AQAAAAAAAAAGAQAAAAMgPCAL", "AAEAAAD/////AQAAAAAAAAAGAQAAAAMgPCAL" } };
             yield return new object[] { "minchar" + char.MinValue + "minchar", new string[] { "AAEAAAD/////AQAAAAAAAAAGAQAAAA9taW5jaGFyAG1pbmNoYXIL", "AAEAAAD/////AQAAAAAAAAAGAQAAAA9taW5jaGFyAG1pbmNoYXIL" } };
 
+            // Exceptions
+            var exception = new Exception("Exception message", new Exception("Inner exception message"))
+            {
+                HelpLink = "http://msdn.microsoft.com",
+                Source = "Exception_Class_Samples",
```

Comment:
Are you missing coverage of HResult field also ? you can set it here to something.

Otherwise it's not verified as both runtimes have the same default.

Reply:
HResult is not settable: `public int HResult { get; protected set; }`. I can of course set it to something via reflection but it has a default value of int.MinValue I think.

Reply:
It sets it to COR_E_EXCEPTION in both sides. I would set it to something by reflection.

Reply:
done.